config/locales/views/dashboard/en.yml

```diff
@@ -111,7 +111,7 @@ en:
           title: This tag has negative follow weight
           text: Anti-follow
         default: Default 1.0
-        empty: You don't follow any tags yet...
+        empty: "Follow tags to personalize your feed. You can explore popular tags %{here}."
```

Comment:
To make this work I think we need to use [safe translations](https://guides.rubyonrails.org/i18n.html#using-safe-html-translations) by adding `_html` to the `empty` key here (and making it match in the view above and also the `fr.yml` translation) would make the link render properly.

**non-blocking suggestion/question:** Perhaps it would make it a bit more natural to read if we only pass in the `link` instead of `here`. That way regardless of the translation language it would be something like this:

```suggestion
        empty_html: "Follow tags to personalize your feed. You can explore popular tags <a href="%{link}>here</a>."
```

What do you think about that? Maybe others have stronger opinions on this though and I'd love to hear them too (if any).

With this change it looks like this to me:

<img width="1320" alt="Screen Shot 2022-11-15 at 20 56 52" src="https://user-images.githubusercontent.com/6045239/202073383-806be299-1fd6-488f-96bc-0ff2aa43d112.png">

The problem with this solution is that it still renders the link ("here") without any separation of the rest of the text. I think this can be fixed by wrapping the translation (either here or in the `following_tags.html.erb` view) with a `<span>` tag so it looks more refined. I gave it a try locally as well and it would end up looking like this:

<img width="1282" alt="Screen Shot 2022-11-15 at 20 54 07" src="https://user-images.githubusercontent.com/6045239/202073490-9ea4c807-5a2c-4703-936e-be2701ee8155.png">

Let me know if you have any questions or alternatives for this 😃